src/datagrid/utils/shortcuts.ts

```diff
@@ -0,0 +1,61 @@
+const allAllCharacters = [
+    'a',
+    'b',
+    'c',
+    'd',
+    'e',
```

Comment:
This cannot work..
it doesn't support special characters.. for example, german and russian characters.